ntbs-service/Models/FilteredSelectLists/FilteredTransferPageSelectLists.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+
+namespace ntbs_service.Models.FilteredSelectLists
+{
+    public class FilteredTransferPageSelectLists
+    {
+        public IEnumerable<OptionValue> TbServices { get; set; }
+        public IEnumerable<OptionValue> CaseManagers { get; set; }

```

Comment:
Yep I actively want to reset CaseManagers to none (so need to supply an empty Case Manager list) and did the logic here instead of in the component i was trying to keep more generic - could change this